ROADMAP.md

```diff
@@ -7,18 +7,18 @@ Each release will include necessary tasks that lead to the completion of the sta
 ### Expanded Completion and Hover
 The `terraform-ls` language server supports basic schema-driven completion. We plan to introduce additional completion and hover capabilities:
 
+- Provide nested navigation symbols (i.e. nested blocks and block attributes) :white_check_mark:
 - Modules
```

Comment:
Given that we prioritized https://github.com/hashicorp/terraform-ls/issues/424 for which a solution is already in progress, I'm not very positive about improving anything module related until Q2.